lib/validate.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'messages'
+require_relative 'ui'
```

Comment:
Nice! I like this Validator class.  Looks like you can remove these dependencies here and in the `initialize`!

Reply:
I agree, this is nice!